src/test/java/com/jcabi/manifests/ManifestsTest.java

```diff
@@ -119,11 +132,103 @@ public void appendsAttributesFromFile() throws Exception {
     public void appendsAttributesFromInputStream() throws Exception {
         final Manifests mfs = new Manifests();
         mfs.append(
-            new StreamsMfs(this.getClass().getResourceAsStream("test.mf"))
+            new StreamsMfs(this.getClass().getResourceAsStream(TEST_MANIFEST))
         );
         MatcherAssert.assertThat(
-            mfs.get("From-File"),
-            Matchers.equalTo("some test attribute")
+            mfs.get(FROM_FILE),
+            Matchers.equalTo(TEST_ATTRIBUTE)
+        );
+    }
+
+    /**
+     * Manifests can read values from several manifests.
+     * @throws Exception If something goes wrong
+     * @since 1.2
+     */
+    @Test
+    public void readsValuesFromSeveralManifests() throws Exception {
+        final List<InputStream> streams = Arrays.asList(
+            this.getClass().getResourceAsStream(TEST_MANIFEST),
+            this.getClass().getResourceAsStream(TEST_MANIFEST_2)
+        );
+        final Collection<String> values = Manifests.all(
+            new StreamsMfs(streams),
+            FROM_FILE
+        );
+        MatcherAssert.assertThat(
+            values,
+            Matchers.hasSize(2)
+        );
+        MatcherAssert.assertThat(
+            values,
+            Matchers.contains(
+                TEST_ATTRIBUTE,
+                    TEST_ATTRIBUTE_2
+            )
+        );
+    }
+
+    /**
+     * Manifests can read values existing in some manifests.
+     * @throws Exception If something goes wrong
+     * @since 1.2
+     */
+    @Test
+    public void readsValuesExistingInSomeManifests() throws Exception {
+        final List<InputStream> streams = Arrays.asList(
+            this.getClass().getResourceAsStream(TEST_MANIFEST),
+            this.getClass().getResourceAsStream(TEST_MANIFEST_2)
+        );
+        final Collection<String> values = Manifests.all(
+            new StreamsMfs(streams),
+            VERSION
+        );
+        MatcherAssert.assertThat(
+            values,
+            Matchers.hasSize(1)
```

Comment:
@nesterov-n please read that: http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#indentation
> you should put as much as possible on one line - within the limit of 80 characters